### ISSUE TITLE
gradle build fails, Manifest merger failed

### ISSUE THREAD

**Issue by ArtworkAD** [ronaldsmartin/Material-ViewPagerIndicator#21]
```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@supportsRtl value=(false) from AndroidManifest.xml:12:9-36
  	is also present at [com.github.ronaldsmartin:Material-ViewPagerIndicator:1.0.2] AndroidManifest.xml:31:9-35 value=(true).
  	Suggestion: add 'tools:replace="android:supportsRtl"' to <application> element at AndroidManifest.xml:6:5-79:19 to override.
```

tools:replace="android:supportsRtl should not be necessary

I created a fix: https://github.com/ronaldsmartin/Material-ViewPagerIndicator/pull/20

**Comment by ronaldsmartin** [comment]
Thanks for this! Will check it out tonight.

**Comment by ronaldsmartin** [comment]
Addressed by #20.
Thanks again for your contribution, @ArtworkAD; will tag a new release this weekend.
